masterlist.yaml

```diff
@@ -16102,6 +16102,19 @@ plugins:
       - crc: 0x1F17FC44
         util: 'SSEEdit v4.0.2f'
 
+  - name: 'Tale of Two Mers.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/43074/'
+        name: 'Tale of Two Mer - An Armor Variant Mod'
+    tag:
+      - Actors.AIData
+      - Delev
+      - Factions
```

Comment:
This appears to be a USSEP forward, I think they just copied the faction to their own ESP to avoid adding USSEP as a master:

```suggestion
```